### ISSUE TITLE
Question: De facto standard?

### ISSUE THREAD

**Issue by joakimskoog** [aspnet/Caching#305]
I'm just wondering if these libraries are the de facto standard solution for caching in a .NET Standard library?  They seem to be a bit heavy and have lots of dependencies for what I'm trying to accomplish but I can't be arsed to roll my own implementations.

**Comment by davidfowl** [comment]
What does de factor standard mean to you? It's a caching abstraction and library written by the ASP.NET team. I don't know what level of "defacto" you're looking for.

**Comment by joakimskoog** [comment]
Fair enough, I guess that the question wasn't great. Let me rephrase myself; is it preferable to use these abstractions on the client side over, say for example, rolling your own more lightweight implementations?

**Comment by davidfowl** [comment]
Honestly, these were made to support ASP.NET applications. Although there are generally applicable, we don't do any extensive testing on client side applications.

**Comment by joakimskoog** [comment]
Ok, sounds good. I'll probably use it anyway (or [CacheManager](https://github.com/MichaCo/CacheManager)) since this [discussion](https://github.com/dotnet/corefx/issues/14529) had some great information.

**Comment by davidfowl** [comment]
👍
